src/Place/Place.php

```diff
@@ -72,6 +73,7 @@
 use DateTimeImmutable;
 use DateTimeInterface;
 use CultuurNet\UDB3\StringLiteral;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language as Udb3Language;
```

Comment:
No need for this extra alias

Reply:
See remark below

Reply:
While in Event the language was indeed already included, this is not the case for place.
Here the default included language is "use CultuurNet\UDB3\Language;"

Reply:
You are correct, for `Place` the `LegacyLanguage` was already imported.